Userland/Applications/Browser/DownloadWidget.cpp

```diff
@@ -168,4 +174,4 @@ void DownloadWidget::did_finish(bool success)
         window()->close();
 }
 
-}
+}
```

Comment:
Please configure your editor to insert line endings, or set up your environment to [automatically run clang-format](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#commit-hooks) and other checks.

That documentation file has also many other goodies, be sure to read it carefully!